### PR TITLE
Added displaying of week days labels github-like feature implementation

### PR DESCRIPTION
**Pull request by andreysaleba**

### CODE REVIEW

File: src/index.jsx [hunk 1]
```diff
@@ -278,8 +331,10 @@ CalendarHeatmap.defaultProps = {
   gutterSize: 1,
   horizontal: true,
   showMonthLabels: true,
+  showWeekdayLabels: true,
```
Comment (okize) [inline src/index.jsx]: should this be `false` by default?

Comment (andreysaleba) [reply src/index.jsx]: Sure, I shouldn't change the old behaviour here.

Comment (andreysaleba) [reply src/index.jsx]: I've updated PR with this change

**Comment by okize** [comment]
published as v1.4.0

sorry for the delay in releasing this @andreysaleba, we really appreciate your efforts on this PR and the others!

**Pull request merged**